pkgs/applications/audio/musly/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, eigen, libav_all }:
+stdenv.mkDerivation rec {
+  pname = "musly";
+  version = "unstable-2017-04-26";
+  src = fetchFromGitHub {
+    owner = "dominikschnitzer";
+    repo = "musly";
+    rev = "f911eacbbe0b39ebe87cb37d0caef09632fa40d6";
+    sha256 = "1q42wvdwy2pac7bhfraqqj2czw7w2m33ms3ifjl8phm7d87i8825";
+  };
+  buildInputs = [ cmake eigen (libav_all.override { vaapiSupport = stdenv.isLinux; }).libav_11 ];
+  preBuild = ''
```

Comment:
This whole attribute is not necessary at all – adding `cmake` to build inputs will do that automatically.